services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminExecutionTask.java

```diff
@@ -519,6 +519,8 @@ private void handleSetStore(UpdateStore message) {
 
     params.setStorageNodeReadQuotaEnabled(message.storageNodeReadQuotaEnabled);
     params.setMinCompactionLagSeconds(message.minCompactionLagSeconds);
+    params.setMaxCompactionLagSeconds(message.maxCompactionLagSeconds);
+    LOGGER.info("max compaction lag ms in AdminExecutionTask: " + message.maxCompactionLagSeconds);
```

Comment:
This is debug log? Should we remove this?

Reply:
Fixed.